docs/sgp40_voc_index/arduino.rst

```diff
@@ -107,6 +107,12 @@ VOC Index is zero
 During the initialization phase of the VOC Index algorithm, which is the first
 100 iterations, the VOC Index will output zero.
 
+Arduino Nano 33 BLE doesn't showup in port list
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Try double tapping the rest button located on the top of the arduino board.
+Otherwise switch USB cables, restart the IDE and reconnect the arduino.
```

Comment:
```suggestion
Try double tapping the reset button located on the top of the Arduino board.
Otherwise switch USB cables, restart the IDE and reconnect the Arduino.
```

Reply:
Also what do you mean by "switch USB cables"? Using a different cable or just unplug and re-plug again?